### PR TITLE
Add 3 list model

### PR DESCRIPTION
**Pull request by Wijantra**
I add `cerificates.py`, `countries.py` and `production_companies.py` in autogen_db_models/imdb/

Three file is data from imdb 2016

### CODE REVIEW

File: autogen_db_models/imdb/certificates.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+class Certificates(Base):
+    __tablename__ = 'certificates'
+
+    title = Column(String)
+    year = Column(Integer)
+    roman = Column(String)
+    type = Column(String)
+    episodeInfo = Column(String)
+    certificate = Column(String)
```
Comment (MaiMee1) [inline autogen_db_models/imdb/certificates.py:14]: There's no primary key for the tables. I don't think a model can not have primary keys. Maybe create our own if the data doesn't have any?

**Review by MaiMee1 — COMMENTED** [review]
I think we should sanitize the data before we store it in the database so that we store the correct data.

File: autogen_db_models/imdb/certificates.py (hunk 1 above)
Comment (Wijantra) [reply autogen_db_models/imdb/certificates.py:14]: Ok i will add primary key

Comment (Wijantra) [reply autogen_db_models/imdb/certificates.py]: I already add primary key. please check it

**Review by MaiMee1 — APPROVED** [review]

**Comment by MaiMee1** [comment]
I added support to insert data and fixed some issues in the parser. Looks OK? @NutthanichN @th-bunratta

**Review by th-bunratta — APPROVED** [review]
Looks good. Good to go!

**Review by NutthanichN — APPROVED** [review]

**Pull request merged**